tools/perl.yaml

```diff
@@ -24,5 +24,8 @@ platforms:
       - /usr/local/bin/perl
       - /usr/bin/perl
   Windows:
+    path_checks:
+      - perl.exe
     file_checks:
       - C:\/Perl64/bin/perl.exe
+      - C:\/Users\/micah\/AppData\/Local\/activestate\/cache\/bin\/perl.exe
```

Comment:
This path doesn't look right

Reply:
Ugh. Whoops. I overlooked that.  Let me see if the path check is sufficient.

Reply:
Yeah this line isn't really needed if perl.exe is in the path. I'll remove it.

```
❯ msl tool check perl
2023-07-19 09:44:38 perl INFO Detecting tool: perl...
2023-07-19 09:44:38 perl INFO   Checking for perl.exe in PATH
2023-07-19 09:44:38 perl INFO     perl.exe found, at: C:\Users\micah\AppData\Local\activestate\cache\bin\perl.exe
2023-07-19 09:44:38 Mussels WARNING     local:perl FOUND.
```